src/evict/evict_page.c

```diff
@@ -729,6 +729,10 @@ __evict_review(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t evict_flags, bool
 
     WT_ASSERT(session, LF_ISSET(WT_REC_VISIBLE_ALL) || F_ISSET(session->txn, WT_TXN_HAS_SNAPSHOT));
 
+    /* Stats to track the number of evictions triggered during checkpoint is in progress. */
```

Comment:
"Stats to track the number of evictions" -> "Stats to track the number of page evictions"
"triggered during checkpoint is in progress" -> "triggered while checkpoint is in progress"

Reply:
Simplified the comment as `Count page evictions in parallel with checkpoint`.